### ISSUE TITLE
The details in generating <PDBid>_nowat.pdb

### ISSUE THREAD

**Issue by SanFran-Me** [gnina/models#28]
In the PDBbind2016 data directory, I see the following files that seems to be used in your training project.
`<PDBid>_nowat.pdb               -- Receptor structure with all HETATOMS removed`

I was wondering how to generate the` nowat.pdb `for` pocket.pdb `files in PDBbind 2019 dataset, and the ChatGPT told me to use:
`for file in *.pdb; do gnina -i "$file" -o "${file%.pdb}_nowat.pdb" --autobox_ligand ""; done`

I am not sure if this is the same way of your project. If so, even though it works, it might have some differences with your dataset. So I am here to ask your methods to generate 'nowat.pdb' in details.

**Comment by dkoes** [comment]
You generate a pdb file without HETATM records by removing all the HETATM records from the file.  This is not something gnina does.  You can use your favorite cheminformatic toolkit or just run `grep -v HETATM`.

**Comment by SanFran-Me** [comment]
> You generate a pdb file without HETATM records by removing all the HETATM records from the file. This is not something gnina does. You can use your favorite cheminformatic toolkit or just run `grep -v HETATM`.

Yeah, `grep -v HETATM` works. But some amino acids modified by phosphorylation will also be deleted（like `T[THR:phosphorylated]`）, how to avoid these occasions when deleting waters and ions in .pdb file ?

**Comment by dkoes** [comment]
Use your favorite cheminformatic tool.  PyMOL, ProDY, etc can all do this sort of thing.  This is outside the scope of gnina.
